src/Imath/ImathVec.h

```diff
@@ -74,9 +75,36 @@ template <class T> class Vec2
     /// Construct from Vec2 of another base type
     template <class S> IMATH_HOSTDEVICE IMATH_CONSTEXPR14 Vec2 (const Vec2<S>& v) noexcept;
 
+    /// Interoperability constructor from another type that behaves as if it
+    /// were an equivalent vector.
+    template<typename V, IMATH_ENABLE_IF(has_xy<V,T>::value)>
+    IMATH_HOSTDEVICE IMATH_CONSTEXPR14 Vec2 (const V& v) noexcept
```

Comment:
Can this be "constexpr" instead of IMATH_CONSTEXPR14?

Reply:
Hmm, maybe. I'll try it out.

Reply:
Yes, the versions that copy from member variables can be just `constexpr`, so updated.